ninja.lua

```diff
@@ -673,16 +679,20 @@ function ninja.generateProjectCfg(cfg)
 	p.outln("ninja_required_version = 1.6")
 	p.outln("")
 
+	local is_c_or_cpp = cfg.language == p.C or cfg.language == p.CPP;
+
 	---------------------------------------------------- figure out settings
-	local pch = nil
-	if toolset ~= p.tools.msc then
-		pch = p.tools.gcc.getpch(cfg)
-		if pch then
-			pch = {
-				input = pch,
-				placeholder = project.getrelative(cfg.workspace, path.join(cfg.objdir, path.getname(pch))),
-				gch = project.getrelative(cfg.workspace, path.join(cfg.objdir, path.getname(pch) .. ".gch"))
-			}
+	if is_c_or_cpp then
```

Comment:
@tritao: 
Not sure if it is that change, but `pch` is now broken in tests:
https://github.com/Jarod42/premake-sample-projects/actions/runs/14692329415/job/41229324622

Reply:
You can either run
`test_projects.py premake5 projects ninja [args]` from root. (run all tests)
or run `premake5 ninja` from `projects/pch` and locally run `ninja` from `projects/pch/solution/ninja`.
Tell if something might be improved from your point of view.